CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ project(Algorithms_in_C++
 # set(CMAKE_CXX_CPPLINT "~/anaconda3/bin/cpplint --filter=-legal/copyright --std=c++11")
 # find_program(CLANG_FORMAT "clang-format")
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Wouldn't we have to update all the other codes to match the C++-17 syntax?

Reply:
I changed the general CMake file; I'll check any other now and update them all

Reply:
Seems there are no others which define their own C++ version.  
Compilation threw a warning from another algorithm, yet it seems that this should've existed before (no return in non-void)

Reply:
So what should we do, then? I'm confused. 🤔

Reply:
> I think the issue is in the CI: could it be that it is set to use C++11?

Hmm, I think it's due to that, not sure though. Maybe we should try to make it work in C++11.